website/addons/wiki/model.py

```diff
@@ -51,7 +54,67 @@ def subscribe_on_write_permissions_revoked(node):
         wiki_utils.migrate_uuid(node, wiki_name)
 
 
-def build_wiki_url(node, label, base, end):
+def wiki_updates(func):
```

Comment:
I'm not sure this decorator is necessary. Wouldn't it be simpler to just use a helper function?

```python
@wiki_deleted.connect
def subscribe_wiki_deleted(name, node, **kwargs):
    # ...
    send_wiki_notification(path=path, message=message)
```